decode_test.go

```diff
@@ -16,6 +16,8 @@ age = 250
 andrew = "gallant"
 kait = "brady"
 now = 1987-07-05T05:45:00Z
+nowCanada = 2017-06-22T02:14:36-06:00
+nowChina = 2017-06-22T16:15:21+08:00
```

Comment:
> It looks like the simple struct below should include a few more fields and answer should do a comparison of these to complete the testing.

Hi Nathan!  Yes, you are absolutely right!  I was lazy and just wanted to get the issue solved in Hugo and ended up not really examining decode_test.go at all.  (really quick and dirty)

Thanks for catching that!  I will make another pull request which will hopefully complete the test.